app/models/audit.rb

```diff
@@ -31,11 +31,11 @@ def before_create
   end
 
   def set_version_number
-    max = Audit.where(auditable_id: auditable_id, auditable_type: auditable_type).reverse(:version).first.try(:version) || 0
+    max = Audit.where(auditable_id:, auditable_type:).reverse(:version).first.try(:version) || 0
```

Comment:
Use the Ruby 3.1 hash literal value syntax when your hash key and value are the same.